xml/art_solution_monitoring.xml

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="file:/usr/share/xml/geekodoc/rng/geekodoc5-flat.rnc"
+   type="application/relax-ng-compact-syntax"?>
+
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<!--
+Further information:
+
+* SAP Monitoring will become easier and powerful using SLES for SAP
+  https://suse.com/c/coming-soon-sap-landscape-monitoring-on-sles-for-sap/
+-->
+
+<article version="5.1" xml:lang="en" xml:id="art-sol-monitoring"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+   <title>&sap; Monitoring</title>
+   <info>
+     <abstract>
+       <para>
+         This article shows monitoring solutions for &sap; administrators
+         to efficiently monitor their &sap; systems.
+       </para>
+       <para>
+         The solution described herein is not officially supported by &suse;.
+         Nevertheless, it is a solution based on best-efforts and published
+         <quote>as is</quote>.
+       </para>
+     </abstract>
+   </info>
+
+  <sect1 xml:id="sec-sol-monit-concept-overview">
+    <title>Conceptual Overview</title>
+    <para>
+      Starting from the idea to improve user experience, &suse; engineering worked
+      on how to monitor and troubleshoot clusters.
+    </para>
+    <para>
+      The exporters showed here can be combined and integrated with Prometheus
```

Comment:
Add a short explanation what 'exporters' are?

Reply:
Rephrased the sentence:

    The exporters showed here export their metrics which can be combined and
    integrated with Prometheus and Grafana to produce complex dashboards.

It's probably not nice, but... it's correct.